### ISSUE TITLE
Database security

### ISSUE THREAD

**Issue by qmada** [UnamSanctam/UnamWebPanel#174]
If my page cant be found then my database is secured? what do i have to do to disable it?

**Comment by UnamSanctam** [comment]
Yes if they cannot access your database file then they won't be able to access it.

**Comment by qmada** [comment]
how do i disable my security?

**Comment by UnamSanctam** [comment]
You mean how to make it so that everyone can access the file? If you're using Apache then you can disable it by editing the `.htaccess` file and removing the whole <FilesMatch> block (line 2 to 4).

**Comment by qmada** [comment]
![image](https://user-images.githubusercontent.com/110357110/182362448-5121c4e3-bc43-45f7-8344-af142538e2e2.png)
I cant open .htaccess file.

**Comment by UnamSanctam** [comment]
Open the `.htaccess` file with a text editor and change it. If it's on a remote webserver then you can first download the `.htaccess` file, then edit it and then upload it again.

**Comment by qmada** [comment]
i did it, nothing changed

**Comment by UnamSanctam** [comment]
Then it might be an IIS server, in that case you can just delete the `web.config` file.

**Comment by qmada** [comment]
deleted it, still nothing

**Comment by UnamSanctam** [comment]
Then you didn't modify the `.htaccess` folder correctly, those two are the only files that limit access to the database file. Unless your webhost itself somehow blocks all .db files from being accessed.

**Comment by UnamSanctam** [comment]
There could also be a .htaccess file inside the `db` folder as well if you didn't edit/remove that, I'm not currently at my computer so I can't check if there is.

**Comment by qmada** [comment]
why cant i load folders into public html?

**Comment by qmada** [comment]
can i just copy files without folders?

**Comment by UnamSanctam** [comment]
It depends on how you upload things, some built-in webhost file managers aren't very good. I would recommend using either FTP or seeing if you can extract .zip files in your file manager. 

But to disable the .db protection first edit the `.htaccess` in the main folder and remove the lines (line 2 to 4), then remove the `web.config` file and then finally remove the `.htaccess` file inside the `db` folder if there is a `.htaccess` there. That should be enough.

**Comment by UnamSanctam** [comment]
Yes all files need to be in their respective folders, but you can usually create folders inside the webhost file manager and copy files into that which was what I meant. Easiest way to upload is usually via FTP or .zip extraction if supported.

**Comment by qmada** [comment]
yeah i extracted them in their respective folders, but it was only working when i put db into publilc_html

**Comment by UnamSanctam** [comment]
Yes everything needs to look like this https://github.com/UnamSanctam/UnamWebPanel/tree/master/UnamWebPanel (folders and files) for it to work.

**Comment by qmada** [comment]
![image](https://user-images.githubusercontent.com/110357110/182424415-6d335651-2657-4f66-83d9-273baddd8c3f.png)
yes everything looks fine, but not working

**Comment by UnamSanctam** [comment]
What isn't working, the webpanel or trying to disable the database security?

**Comment by UnamSanctam** [comment]
You mean mysite.com/db/unamwebpanel.db? Is it still forbidden?

**Comment by qmada** [comment]
oh, yeah i forgot that i put it back into the folder. Its working now its just a white screen

**Comment by qmada** [comment]
cant see the login screen when i go to my site just a white screen. What should i do? PHP version is 8

**Comment by UnamSanctam** [comment]
Well your file sizes don't look correct in your latest screenshot, open up the files in the file manager there and see what they contain.

**Comment by qmada** [comment]
i have an error and i cant find logs in the unam lib folder. chrome says my site is deceptive

**Comment by UnamSanctam** [comment]
Which error? And Chrome "malicious/deceptive" site detection is based on URLs usually, so your URL has been scanned/reported and placed into a blacklist. Though that won't affect the miner in any way.

**Comment by qmada** [comment]
application error

**Comment by UnamSanctam** [comment]
Then it sounds like not all files have been uploaded correctly.

**Comment by qmada** [comment]
![image](https://user-images.githubusercontent.com/110357110/182610967-3e50efa6-feaf-4d07-af29-1dfee4f4590d.png)
Everything seems to be on the place

**Comment by UnamSanctam** [comment]
Then it should be working, for 000webhost (which it looks like you might be using) then all you need to do for it to work is to 
1) Change PHP version to 7.4 or above 
2) Upload all the files
Then everything will work.

**Comment by qmada** [comment]
yes, ty. When i copied all these files the original .htaccess file was missing
